CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.0.0
```

Comment:
I'd make it a bugfix version personally.

Reply:
I am worried that key changes my break somebody, basically it's not just a fix in matching but keys with dots will now be with dashes, so if someone relied on this they will have to make changes on their side. Hence major bump, but up to you.

Reply:
Any bug fix is a breaking change for someone. Let's make it to minor I'm fine breaking some probably a small percentage of users.

Reply:
Changed to 2.7.0, or would you prefer 2.6.1? @gaborbernat

Reply:
Is fine as is.